crest/Assets/Crest/Crest/Scripts/LodData/RegisterLodDataInput.cs

```diff
@@ -73,7 +91,7 @@ public void Draw(CommandBuffer buf, float weight, int isTransition, int lodIdx)
         static void InitStatics()
         {
             // Init here from 2019.3 onwards
-            _registrar = new Dictionary<System.Type, List<ILodDataInput>>();
+            _registrar = new Dictionary<System.Type, OceanInput>();
```

Comment:
You should be able to replace this with `.Clear()`. Even without static domain reload it won't skip static initialisation for the very first time.

Reply:
Thanks will do